snuba/datasets/transactions_processor.py

```diff
@@ -248,6 +248,12 @@ def _process_contexts_and_user(
             if context in contexts:
                 del contexts[context]
 
+        appContext = contexts.get("app")
```

Comment:
Is there a reason these variables are camel cased?

Reply:
Ups, sorry no. I'm used to camel casing in Swift, ObjC, Java, ... I'm going to change that.